metaflow/lint.py

```diff
@@ -31,6 +31,8 @@ def ensure_acyclicity(self, f):
     def ensure_non_nested_foreach(self, f):
         return self._decorate("require_non_nested_foreach", f)
 
+    # TODO: cluster_size matches with a multinode
```

Comment:
we should implement this :) Now `cluster_size` fails strangely if `@multinode` is not present

Reply:
Yep, was thinking of doing it in separate PR, but can do it here as well.